doc/flame/other/util.md

```diff
@@ -195,6 +195,11 @@ Methods:
  - `toVertices`: Turns the four corners of the `Rect` into a list of `Vector2`.
  - `toMathRectangle`: Converts this `Rect` into a `math.Rectangle`.
  - `toGeometryRectangle`: Converts this `Rect` into a `Rectangle` from flame-geom.
+ - `transform`: Transforms the `Rect` using a `Matrix4`.
+
+Factories:
```

Comment:
Maybe clarify that you have to write `RectExtension.` to use these ones?

Reply:
We don't have that on the other docs either, should I add it there as well for each extension?

Reply:
Added them.